docs/4.3/openssh_teleport.md

```diff
@@ -0,0 +1,311 @@
+```
+
+If you have multiple Teleport clusters, you have to export and set up these
+certificate authorities for each cluster individually.
+
+!!! tip "OpenSSH and Trusted Clusters"
+
+    If you use [recording proxy mode](architecture/teleport_proxy.md) and [trusted
+    clusters](trusted-clusters.md), you need to set up certificate authority from
+    the _root_ cluster to match **all** nodes, even those that belong to _leaf_
+    clusters. For example, if your node naming scheme is `*.root.example.com`,
+    `*.leaf1.example.com`, `*.leaf2.example.com`, then the
+    `@certificate-authority` entry should match `*.example.com` and use the CA
+    from the root auth server only.
+
+Make sure you are running OpenSSH's `ssh-agent` , and have logged in to the
+Teleport proxy:
+
+``` bash
+$ eval `ssh-agent`
+$ tsh --proxy=work.example.com login
+```
+
+`ssh-agent` will print environment variables into the console. Either `eval` the
+output as in the example above, or copy and paste the output into the shell you
+will be using to connect to a Teleport node. The output exports the
+`SSH_AUTH_SOCK` and `SSH_AGENT_PID` environment variables that allow OpenSSH
+clients to find the SSH agent.
+
+Lastly, configure the OpenSSH client to use the Teleport proxy when connecting
+to nodes with matching names. Edit `~/.ssh/config` for your user or
+`/etc/ssh/ssh_config` for global changes:
+
+``` bash
+# work.example.com is the jump host (proxy). credentials will be obtained from the
+# openssh agent.
+Host work.example.com
+    HostName 192.168.1.2
+    Port 3023
+
+# connect to nodes in the work.example.com cluster through the jump
+# host (proxy) using the same. credentials will be obtained from the
+# openssh agent.
+Host *.work.example.com
+    HostName %h
+    Port 3022
+    ProxyCommand ssh -p 3023 %r@work.example.com -s proxy:%h:%p
+
+# when connecting to a node within a trusted cluster with name "remote-cluster",
+# add the name of the cluster to the invocation of the proxy subsystem.
+Host *.remote-cluster.example.com
+   HostName %h
+   Port 3022
+   ProxyCommand ssh -p 3023 %r@work.example.com -s proxy:%h:%p@remote-cluster
+```
+
+When everything is configured properly, you can use ssh to connect to any node
+behind `work.example.com` :
+
+``` bash
+$ ssh root@database.work.example.com
+```
+
+!!! tip "Note"
+
+    Teleport uses OpenSSH certificates instead of keys which means
+    you can not connect to a Teleport node by IP address. You have to connect by
+    DNS name. This is because OpenSSH ensures the DNS name of the node you are
+    connecting is listed under the `Principals` section of the OpenSSH
+    certificate to verify you are connecting to the correct node.
+
+
+To connect to the OpenSSH server via `tsh`, add `--port=<ssh port>` with the `tsh ssh` command:
+
+Example ssh to `database.work.example.com` as `root` with a OpenSSH server on port 22 via `tsh`:
+
+```bash
+tsh ssh --port=22 root@database.work.example.com
```

Comment:
```suggestion
tsh ssh --port=22 dev@database.root.example.com
```

Doing this to continue the `root.example.com` theme and changing the user so we don't repeat `root`.